ui/pages/create-account/new-account.stories.js

```diff
@@ -15,7 +16,12 @@ export default {
 };
 export const DefaultStory = (args) => {
   return (
-    <NewAccountCreateForm {...args} createAccount={action('Account Created')} />
+    <Box className="new-account">
+      <NewAccountCreateForm
+        {...args}
+        createAccount={action('Account Created')}
+      />
+    </Box>
```

Comment:
We can remove Box here
```suggestion
      <NewAccountCreateForm
        {...args}
        createAccount={action('Account Created')}
      />
```